src/test/java/com/google/gcp_variant_transforms/library/VariantToBqUtilsTest.java

```diff
@@ -0,0 +1,248 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.library;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.google.api.services.bigquery.model.TableRow;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypesContext;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFFormatHeaderLine;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFHeaderLineCount;
+import htsjdk.variant.vcf.VCFHeaderLineType;
+import htsjdk.variant.vcf.VCFInfoHeaderLine;
+import org.junit.Before;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Units tests for VariantToBqUtils.java
+ */
+public class VariantToBqUtilsTest {
+  private static final String ALTERNATE_BASES_ALT = "alt";
+  private static final String CALLS_NAME = "name";
+  private static final String CALLS_GENOTYPE = "genotype";
+  private static final String CALLS_PHASESET = "phaseset";
+  private static final String DEFAULT_PHASESET = "*";
+  private static final String MISSING_FIELD_VALUE = ".";
+
+  VariantContext variantContext;
+  GenotypesContext genotypesContext;
+  VCFHeader vcfHeader;
+
+  @Before
+  public void constructMockClasses() {
+    variantContext = mock(VariantContext.class);
+    vcfHeader = mock(VCFHeader.class);
+    genotypesContext = mock(GenotypesContext.class);
+  }
+
+  @Test
+  public void testConvertStringValueToRightValueType_whenComparingElement_thenTrue() {
+    // test list of values
+    String integerListStr = "23,27";
```

Comment:
I suggest arranging both lists before the asserts. I'm wondering if it would be good to go even further and arrange all the variables in this test function before all the asserts.

Reply:
I have moved other field or values to constants in the test, but in this method I think adding custom input value directly in the function will be more clear about the change of the value type. Put these value to static constants naming looks a bit hard to find the value change.